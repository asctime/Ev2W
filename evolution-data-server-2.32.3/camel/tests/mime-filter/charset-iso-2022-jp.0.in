えろいｔｙうとうぃういえｙてｒちうえｒｙとｙｒとえつお３９４５０９８０９４えいｒ
とえつおえうとえるとうｔ

Thanks & Regards,
Sures
