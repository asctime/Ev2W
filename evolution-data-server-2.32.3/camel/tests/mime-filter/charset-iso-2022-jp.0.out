えろいｔｙうとうぃういえｙてｒちうえｒｙとｙｒとえつお３９４５０９８０９４えいｒ
とえつおえうとえるとうｔ

Thanks & Regards,
Sures
